docs/recipes.md

```diff
@@ -179,7 +179,7 @@ for path in sorted(Path("src").rglob("*.py")):
     elif parts[-1] == "__main__":
         continue
 
-    nav[parts] = str(doc_path)  # (1)
+    nav[parts] = str(PurePosixPath(doc_path))  # (1)
```

Comment:
```suggestion
    nav[parts] = doc_path.as_posix()  # (1)
```

Please double-check that my suggestion actually works and then please apply this style instead

Reply:
@oprypin Nice suggestion! I didn't know about `as_posix()`. That is certainly a cleaner way to get the same outcome.